README.md

```diff
@@ -22,17 +21,20 @@ IPLD is a set of standards and implementations for creating decentralized data-s
 - [Contribute](#contribute)
 - [License](#license)
 
-## Enter IPLD
+## Docs
 
-**Watch the Merkle Forest Talk**
+- [IPLD Specs](https://github.com/ipld/specs)
+- [CID spec](https://github.com/multiformats/cid)
 
-[![](/img/enter-merkle-forest.jpg)](https://www.youtube.com/watch?v=Bqs_LzBjQyk)
+## Stacks
 
-## Docs
+IPLD has a [layer model]() and is much more than "one thing." As such, implementations of IPLD aren't single libraries but stacks of libraries that vary a bit by programming language.
```

Comment:
There's the actual link target missing.

Reply:
yup, want to work out the structure before filling in all the details.